Paper.tex

```diff
@@ -2785,7 +2785,9 @@ \subsection{Instruction Set}
 \midrule
 0xfe & {\small INVALID} & $\varnothing$ & $\varnothing$ & Designated invalid instruction. \\
 \midrule
-\linkdest{selfdestruct}{}0xff & {\small SELFDESTRUCT} & 1 & 0 & Halt execution and register account for later deletion. \\
+\linkdest{selfdestruct}{}0xff & {\small SELFDESTRUCT} & 1 & 0 & Halt execution and register account for later deletion. Readers should be warned, \\
+&&&& since the \textit{Shanghai} update, this opcode has been marked as deprecated. \\
+&&&& Its behavior might be subject to change in an upcoming update. \\
```

Comment:
Suggesting adding that it may be subject "to **breaking** changes"

Reply:
changes in [91f0547](https://github.com/ethereum/yellowpaper/pull/906/commits/91f054787b4f38d15bddcefa09bc98ea0a62144b)